besser/bot/platforms/platform.py

```diff
@@ -61,3 +61,5 @@ def reply(self, session: 'Session', message: str) -> None:
             message (str): the message to send to the user
         """
         pass
+    
```

Comment:
Note: We should always leave 1 empty line at the end of each python module (not 0 nor 2)